### PR TITLE
Nexus 4674 - Reindexing one GAV of a GA removes other versions from index

### PR DESCRIPTION
**Pull request by velo**
Basically bump indexer version and add IT.

The real fix is at
https://github.com/sonatype/maven-indexer/commit/78eab0db4790aabcc489411aaad3d855702b13f5

Must wait maven-indexer to be updated before merging this.

### CODE REVIEW

**Comment by peterlynch** [comment]
+1 when maven-indexer updated

**Comment by velo** [comment]
Apache CI just built my changes, and it is happy
https://builds.apache.org/job/maven-indexer/44/

Thanks @cstamas

**Comment by velo** [comment]
our CI is happy about this all well
https://builds.sonatype.org/view/nexus/job/nexus-oss-its-feature/99/
+1

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4674/Nexus4674UpdateGAVIndexIT.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    extends AbstractNexusIntegrationTest
+{
+
+    @Test
+    public void searchTest()
+        throws Exception
+    {
+        // auto deploy artifacts need time to be indexed
+        TaskScheduleUtil.waitForAllTasksToStop();
+        getEventInspectorsUtil().waitForCalmPeriod();
```
Comment (nabcos) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4674/Nexus4674UpdateGAVIndexIT.java:47]: AbstractNexusIntegrationTest is doing waitForCalmPeriod as well, maybe we should also add waitForAllTasks there so we don't need to redo this over and over again

**Comment by nabcos** [comment]
+1

**Comment by cstamas** [comment]
+1

**Pull request merged**